### PR TITLE
fix  #135

### LINKED ISSUE #135: Drop item bug
How to:
in order u need to use Classic GUI and players need to have items in his inventory
1-join with player name (Do not login)
2- put items in Crafting area 
3- quit mcpe 

rejoin and u will see that u did lose ur items and items will be droped

### PR DESCRIPTION
**Pull request by VCraftMCPE**

### CODE REVIEW

**Comment by VCraftMCPE** [comment]
this will fix it and create another issue for not receiving items after login

File: ServerAuth/src/ServerAuth/EventListener.php [hunk 1]
```diff
@@ -185,6 +187,18 @@ public function onCraftItem(CraftItemEvent $event) {
             }
         }
     }
+       /**
```
Comment (inxomnyaa) [inline ServerAuth/src/ServerAuth/EventListener.php]: intendation, empty line

**Comment by VCraftMCPE** [comment]
that should fix it

**Comment by VCraftMCPE** [comment]
$transactions = $event->getTransaction()->getTransactions();
		foreach($transactions as $transaction){
			if($transaction->getInventory()->getHolder() instanceof Player and !ServerAuth::getAPI()->isPlayerAuthenticated($transaction->getInventory()->getHolder())){
						$event->setCancelled();

**Comment by VCraftMCPE** [comment]
i cant edit it bc i delete the fork

**Review by inxomnyaa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**